docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -248,7 +212,6 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
               $defaults['layout_builder_style_card_image_size'] = 'card_image_medium';
               break;
 
-            case 'views_block:events_list_block-card_list':
```

Comment:
Was removing `case 'views_block:events_list_block-card_list':` intentional?

Reply:
It was a duplicate line. I left in the one that was grouped with the block listing events from content hub.